src/parser.rs

```diff
@@ -829,11 +872,16 @@ mod tests {
             resolved_value: "The Rolling Stones".to_string(),
             raw_value: "the stones".to_string(),
         });
+
+        let license_filename = "LICENSE".to_string();
+        let license_content = "Some license content\nIn Here".to_string();
+
         let parser = ParserBuilder::default()
             .minimum_tokens_ratio(0.5)
             .gazetteer(gazetteer)
             .n_stop_words(2)
             .additional_stop_words(vec!["hello".to_string()])
+            .license_info(license_filename, license_content)
```

Comment:
In this test, you should check that:
- the persisted parser metadata contains the correct license info
- the reloaded parser contains the correct license info